docs/contribute/documentation.md

```diff
@@ -0,0 +1,51 @@
+---
+html_theme.sidebar_secondary.remove: true
+---
+
+# 📝 Documentation
+
+The primary objective of Bookshelf is to streamline the datapack development process and ensure accessibility. To achieve this, it is crucial that the library is thoroughly documented.
+We have dedicated significant effort to creating this comprehensive documentation website.
+Moreover, we request that every new feature contribution be accompanied by appropriate documentation.
+
+This page aims to give some information on how to contribute to the documentation itself.
+
+## ⚙️ Install the required tools
+
+Bookshelf documentation requires Python to be built.
+If you are not familiar with this programming language, don't worry, you don't have to use it.
+But to be able to build locally the documentation locally, you will still need to install it.
+For that, you can download it on the [Python website](https://www.python.org/downloads/).
+We recommend the Python version 3.12.
+After having installed Python and cloned the Bookshelf repository, go on the `/docs` folder and open a terminal (or Powershell if you are on Windows).
+Then, execute this command:
+
+```shell
+pip install -r requirements.txt
```

Comment:
maybe `pip install -r requirements.txt -U`? This tell to also upgrade which is useful when sphinx or the theme get updated